posthog-core/src/types.ts

```diff
@@ -231,7 +234,7 @@ export type PostHogFlagsAndPayloadsResponse = Partial<
   Pick<PostHogDecideResponse, 'featureFlags' | 'featureFlagPayloads'>
 >
 
-export type JsonType = string | number | boolean | null | { [key: string]: JsonType } | Array<JsonType>
+export type JsonType = string | number | boolean | null | { [key: string]: JsonType } | Array<JsonType> | JsonType[]
```

Comment:
sometimes only `Array<JsonType>` did not work and fixing everywhere was way more changes

Reply:
I hate this whole `Array<T>` vs `T[]` thing, and I agree this is fine